.env_sample

```diff
@@ -1,7 +1,6 @@
 Create a file named .env in the root of this directory.  
 Afterwards create the following environment variables:
 
-GMAIL_DOMAIN=YOUR EMAIL DOMAIN HERE
-GMAIL_USERNAME=YOUR EMAIL USERNAME HERE
-GMAIL_PASSWORD=YOUR EMAIL PASSWORD HERE
-SUCCESS_URL=YOUR SUCCESS URL HERE
\ No newline at end of file
+GMAIL_DOMAIN=gmail.com
+GMAIL_USERNAME=vala.app.backend@gmail.com
+GMAIL_PASSWORD=valaapp1234
```

Comment:
Are these real passwords? `.env-sample` is a template. You should never commit real passwords to git!!!